acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -56,132 +56,6 @@ def install_packages_on(hosts, package_hash, options = {})
         return true
       end
 
-      def fetch(base_url, file_name, dst_dir)
```

Comment:
It looks like you're also going to want to delete the corresponding tests from `lib/puppet/acceptance/install_utils_spec.rb`